src/status_im2/common/data_store/wallet.cljs

```diff
@@ -15,6 +15,11 @@
   [ids]
   (string/join constants/chain-id-separator ids))
 
+(defn add-keys-to-account
+  [account]
+  (-> account
+      (assoc :watch-only? (= (:type account) :watch))))
```

Comment:
I usually suggest avoiding threading macros for only one step. I think they worth using when we have at least two steps,